src/overlays/actors/ovl_En_Ossan/z_en_ossan.c

```diff
@@ -1342,7 +1342,7 @@ void EnOssan_InitPartTimeWorker(EnOssan* this, GlobalContext* globalCtx) {
 
 s32 EnOssan_GetWelcomeCuriosityShopMan(EnOssan* this, GlobalContext* globalCtx) {
     Player* player = PLAYER;
-    u16 ret = func_800F1250(globalCtx, 0x2F);
+    u16 ret = Text_GetFaceReaction(globalCtx, 0x2F);
```

Comment:
I realize this isn't an En_Ossan PR, but it might be nice to rename "ret" to faceReaction or something similar, since you did already rename the function.

Reply:
Fair